config/default.yml

```diff
@@ -3549,6 +3549,16 @@ Style/MutableConstant:
   Description: 'Do not assign mutable objects to constants.'
   Enabled: true
   VersionAdded: '0.34'
```

Comment:
Please add a `VersionChanged` entry with the next minor version. :bowing_man:

Reply:
Sorry after merging 💦 `VersionChanged` may be unnecessary because the default behavior does not change.

Reply:
True. However, the cop is now configurable. So technically that is a change to the user.

Reply:
Ah, I missed that new config is also target to `VersionChanged`.

> * `VersionChanged` - the RuboCop version in which it was changed in a user-impacting way (new config, updated defaults, etc)

https://github.com/rubocop-hq/rubocop/issues/5978